.travis.yml

```diff
@@ -7,11 +7,15 @@ env:
     - GRADLE_OPTS="-Xmx2048m"
     # wait up to 10 minutes for adb to connect to emulator
     - ADB_INSTALL_TIMEOUT=10
+    # FABRIC_API_SECRET={API Key}
+    - secure: QmSJoNcdg6djHWmtACAjWcANnvibicZFwu8EDYw6d3BTV9s5q8IFOh4ay6jFYjBKWKUNbdMwqn43Z9MBZTB+G4z0hCuemxdETX5yXKNsoQRORzeNZCzWmcWjAEy2ZfvccDoTINai+Gz1ckFiWUV++ONK7NPzPE5zeuQA4kPKb9w=
+    # BETA_KEYSTORE_PASSWORD={password}
+    - secure: ZZREqPHD1TMUgfHUMws3EVy1f0q/qSkdcowSEXb4S//EGolDgkDCL0oWAz9TzKe6RGjj+PQBzY66JgCC5UJ7gT+RxOgSetQwuR+thQ/bW6P1zH5PmiAxfB28jBXC2OpTX7Ad7G2iOjMMFj4WwmfKX/dPP8/WUGhWtGPaF/oEfms=
   matrix:
     - ANDROID_TARGET=android-15 ANDROID_ABI=armeabi-v7a
 #    - ANDROID_TARGET=android-21 ANDROID_ABI=armeabi-v7a
 #    - ANDROID_TARGET=android-23 ANDROID_ABI=armeabi-v7a
-    - ANDROID_TARGET=android-24 ANDROID_ABI=armeabi-v7a
+    - ANDROID_TARGET=android-24 ANDROID_ABI=armeabi-v7a DEPLOY_BETA=true
```

Comment:
any reason to not build on android-26? This is now two versions behind, I think.

Reply:
it's actually 3 versions behind. But, the arm test images are only available up through android-24 the last time I checked.

Reply:
Ah, okay. Can you check and bump it if the later ones are available?

Reply:
there is a google apis ARM system image for android-25. I'd have to update some of the build script to support using that variant, I'll work that up in another PR.

system images for newer versions of android are x86 based, but those images don't work in travis due to the travis architecture.

Reply:
sounds good